flink-connectors/flink-connector-kinesis/src/test/java/org/apache/flink/streaming/connectors/kinesis/manualtests/ManualExactlyOnceWithStreamReshardingTest.java

```diff
@@ -89,15 +90,17 @@ public static void main(String[] args) throws Exception {
 		}
 
 		final Configuration flinkConfig = new Configuration();
-		flinkConfig.setInteger(ConfigConstants.LOCAL_NUMBER_TASK_MANAGER, 1);
-		flinkConfig.setInteger(TaskManagerOptions.NUM_TASK_SLOTS, 8);
 		flinkConfig.setString(TaskManagerOptions.MANAGED_MEMORY_SIZE, "16m");
 		flinkConfig.setString(ConfigConstants.RESTART_STRATEGY_FIXED_DELAY_DELAY, "0 s");
 
-		LocalFlinkMiniCluster flink = new LocalFlinkMiniCluster(flinkConfig, false);
-		flink.start();
+		MiniClusterResource flink = new MiniClusterResource(new MiniClusterResourceConfiguration.Builder()
+			.setNumberTaskManagers(1)
+			.setNumberSlotsPerTaskManager(8)
+			.setConfiguration(flinkConfig)
```

Comment:
It was "new LocalFlinkMiniCluster(flinkConfig, **false**)" which used different Actor systems for job and task managers. 

Is it needed to set here also `.setRpcServiceSharing(RpcServiceSharing.DEDICATED)` as `RpcServiceSharing.SHARED` will be used by default in `MiniClusterResourceConfiguration.Builder` (in both modified tests)?

Reply:
I don't believe this to be relevant for the test. It seems primarily about interactions between the kinesis connector and the outside.

Reply:
Probably, true, I was just thinking to leave it as it was.
`LocalFlinkMiniCluster` also has a constructor where it is `shared` by default but it was overwritten to `dedicated`.